data/core/macros/weapon_specials.cfg

```diff
@@ -151,7 +161,7 @@
     [damage]
         id=absorb
         name= _ "absorb"
-        description= _ "This attack puts the unit in good defensive position, and it absorbs some of the damage dealt by an enemy strike."
+        description= _ "This attack puts the unit in good defensive position, and it absorbs"+" {ATM}%"+ _" damage dealt by an enemy strike."
```

Comment:
Is there any precedence for this in mainline? I don’t think we want to subject translations to this.